Source/WebInspectorUI/UserInterface/Views/TimelineOverview.js

```diff
@@ -563,6 +568,9 @@ WI.TimelineOverview = class TimelineOverview extends WI.View
         // Require twice the vertical delta to overcome horizontal scrolling. This prevents most
         // cases of inadvertent zooming for slightly diagonal scrolls.
         if (Math.abs(event.deltaX) >= Math.abs(event.deltaY) * 0.5) {
```

Comment:
i think we may need to revisit this idea as now a slightly diagonal scroll will cause the entire UI to shift (as well as the overview graph to zoom), which is not very ideal

Reply:
Yeah, the scroll-hijacking isn't great here. The horizontal scroll is artificial and the experience isn't smooth, either with or without this patch. 

Could we consider removing the scroll-hijacking completely and have a dedicate UI control for adjusting the time resolution? I know that's an expected gesture for users of this tab so I'm not sure if we'd be degrading the experience for them.

Reply:
i wont discount the possibility of some other interaction being better than scrolling, but i would be surprised if that exists

i think in this case we may just want to tweak the numbers here (or have some sort of "are we in the middle of a vertical scroll" flag) that just makes it harder to diagonally scroll